test/test_network_service.py

```diff
@@ -166,10 +166,10 @@ def _cleanup(net, subnet, router):
                                        self.provider, 'placement'))
 
             # Check basic router properties
-            self.assertIn(
-                router, self.provider.network.routers(),
-                "Router {0} should exist in the router list {1}.".format(
-                    router.id, self.provider.network.routers()))
+            # self.assertIn(
```

Comment:
@afgane What do you think about this? Should we do a conditional skip for Azure? Or comment out the entire test?